dev-packages/application-manager/src/application-package-manager.ts

```diff
@@ -89,9 +90,9 @@ export class ApplicationPackageManager {
             const inspectArg = mainArgs.splice(inspectIndex, 1)[0];
             options.execArgv = ['--nolazy', inspectArg];
         }
-        return this.__process.bunyan(
-            this.__process.fork(this.pck.backend('main.js'), mainArgs, options)
-        );
+        const child = this.__process.fork(this.pck.backend('main.js'), mainArgs, options);
```

Comment:
here, `stdio: [0, 1, 2, 'ipc']` should do instead of manual piping

Reply:
👍